### ISSUE TITLE
python310Packages.mxnet fails

### ISSUE THREAD

**Issue by fabaff** [NixOS/nixpkgs#174872]
### Describe the bug
Rebuilding the Python bindings of `mxnet` fails.

### Steps To Reproduce

```bash
...nixpkgs:~/nixpkgs]$ nix-build -A python310Packages.mxnet --check
copying path '/nix/store/alxz2yvl7yg4pqq1l0pzhshnxhfk7lbx-apache-mxnet-src-1.8.0-incubating.tar.gz' from 'https://cache.nixos.org'...
checking outputs of '/nix/store/nd1rj4s9ad1ndj1cqycr6lrfmxbdrln4-python3.10-mxnet-1.8.0.drv'...
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing setuptools-build-hook
Using setuptoolsBuildPhase
Using setuptoolsShellHook
Sourcing pip-install-hook
Using pipInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
unpacking sources
unpacking source archive /nix/store/alxz2yvl7yg4pqq1l0pzhshnxhfk7lbx-apache-mxnet-src-1.8.0-incubating.tar.gz
source root is apache-mxnet-src-1.8.0-incubating
setting SOURCE_DATE_EPOCH to timestamp 1611092364 of file apache-mxnet-src-1.8.0-incubating/3rdparty/tvm/3rdparty/rang/test_package/visualTest.cpp
patching sources
configuring
no configure script, doing nothing
building
Executing setuptoolsBuildPhase
/nix/store/h8l770plqx6aryc579yc8l55gs11csrc-setuptools-setup-hook/nix-support/setup-hook: line 20:    27 Illegal instruction     (core dumped) /nix/store/cc0wyzb5d4pxa80gm4bk8abkjr4i2kw5-python3-3.10.4/bin/python3.10 nix_run_setup bdist_wheel
```

### Expected behavior

Run passes

### Additional context

Perhaps a simple update to 1.9.1 solves the issue.

### Notify maintainers

@abbradar

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
...nixpkgs:~/nixpkgs]$ nix-shell -p nix-info --run "nix-info -m"
this path will be fetched (0.06 MiB download, 0.30 MiB unpacked):
  /nix/store/gvvw3shxbymvwqqc8z2zb84bh5dbzn99-bash-interactive-5.1-p16-dev
copying path '/nix/store/gvvw3shxbymvwqqc8z2zb84bh5dbzn99-bash-interactive-5.1-p16-dev' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.115, NixOS, 22.05 (Quokka), 22.05pre378655.48037fd9042`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.1`
 - channels(root): `"nixos-22.05pre378655.48037fd9042"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by totoroot** [comment]
@fabaff This can be closed, right? mxnet-1.9.1 builds fine.

**Comment by fabaff** [comment]
> mxnet-1.9.1 builds fine.

Yes, it builds.
